plasma_framework/test/helpers/transaction.js

```diff
@@ -1,24 +1,24 @@
 const rlp = require('rlp');
-
 const { BN } = require('openzeppelin-test-helpers');
+const { EMPTY_BYTES_32, OUTPUT_TYPE } = require('../helpers/constants.js');
 
-const EMPTY_BYTES32 = `0x${Array(64).fill(0).join('')}`;
 const TransactionTypes = {
     PLASMA_DEPOSIT: 1,
 };
 
 class PaymentTransactionOutput {
-    constructor(amount, owner, token) {
+    constructor(amount, owner, token, type = OUTPUT_TYPE.PAYMENT) {
```

Comment:
minor: might be better and more intuitive to use if we put the args with the same order of the output. so `type, owner, token, amount`?

Reply:
yeah, I agree... and I started that way, but then took the easy way out above because it meant much less changes to the existing tests 😂 
But I'll go back and make the changes - it is better to have the correct order